jobs/ci-kubernetes-e2e-gce-cos-beta-k8s-current-serial.env

```diff
@@ -0,0 +1 @@
+PROJECT=k8s-e2e-cos-beta-k8s-current-serial
```

Comment:
This is really annoying, but I'd like to rename the set of channels we're using for k8s.

Instead of {dev,new,stable,old,veryold} I'd like to use {dev,stable1,stable2,stable3,stable4}. I think that matches people's thinking a lot better, and helps underline that they're all stable branches.